src/api/app/models/user.rb

```diff
@@ -32,6 +32,10 @@ class User < ApplicationRecord
   has_many :groups_users, inverse_of: :user
   has_many :roles_users, inverse_of: :user
   has_many :relationships, inverse_of: :user, dependent: :destroy
+  has_many :direct_maintained_project_relationships, -> { projects.where(role_id: Role.hashed['maintainer'].id) }, class_name: 'Relationship'
```

Comment:
There is a scope in Relationships
``
  scope :projects, -> { where("project_id is not null") }
``